### PR TITLE
Updated documentation for Line Continuation Comments and added VB16 V…

### PR DESCRIPTION
**Pull request by paul1956**
…S 2019 headings. There may be other VB16 features

## Summary
Added documentation for Comment after Line Continuation which is new in VB 16
Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by rpetrusha** [comment]
@paul1956, there's a merge conflict; could you resolve it, please. Also, your *.gitignore* file should not be included in your commit.

File: docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ When writing your code, you might at times create lengthy statements that necess
   
 ### To break a single statement into multiple lines  
   
--   Use the line-continuation character, which is an underscore (`_`), at the point at which you want the line to break. The underscore must be immediately preceded by a space and immediately followed by a line terminator (carriage return).  
+-   Use the line-continuation character, which is an underscore (`_`), at the point at which you want the line to break. The underscore must be immediately preceded by a space and immediately followed by a line terminator (carriage return) or starting with language vesion 16 a comment is allowed after a line continuation..
```
Comment (Thraka) [inline docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md]:
```suggestion
-   Use the line-continuation character, which is an underscore (`_`), at the point at which you want the line to break. The underscore must be immediately preceded by a space and immediately followed by a line terminator (carriage return) or starting with language version 16 a comment is allowed after a line continuation..  
```

File: docs/visual-basic/getting-started/whats-new.md [hunk 2]
```diff
@@ -51,6 +53,12 @@ Bit-shift operators, loop variable declaration
 
 Visual Basic / Visual Studio .NET 2002   
 The first release of Visual Basic .NET
+## Visual Basic 16
```
Comment (Thraka) [inline docs/visual-basic/getting-started/whats-new.md]: Adding spacing around headers.

```suggestion

## Visual Basic 16

```

File: docs/visual-basic/getting-started/whats-new.md [hunk 3]
```diff
@@ -16,11 +16,13 @@ ms.author: ronpet
...
   
 ## Current version
+Visual Basic 16 / Visual Studio 2019 Version 16.0  
+For new features, see [Visual Basic 16](#visual-basic-16)
 
-Visual Basic 15.5 / Visual Studio 2017 Version 15.5  
-For new features, see [Visual Basic 15.5](#visual-basic-155)
 
 ## Previous versions
+Visual Basic 15.5 / Visual Studio 2017 Version 15.5
```
Comment (Thraka) [inline docs/visual-basic/getting-started/whats-new.md]: Adding spacing around header
```suggestion

Visual Basic 15.5 / Visual Studio 2017 Version 15.5  
```

File: docs/visual-basic/getting-started/whats-new.md [hunk 4]
```diff
@@ -16,11 +16,13 @@ ms.author: ronpet
 This topic lists key feature names for each version of Visual Basic, with detailed descriptions of the new and enhanced features in the lastest version of the language.
   
 ## Current version
+Visual Basic 16 / Visual Studio 2019 Version 16.0
```
Comment (Thraka) [inline docs/visual-basic/getting-started/whats-new.md]: Add spacing around header
```suggestion

Visual Basic 16 / Visual Studio 2019 Version 16.0  
```

**Review by Thraka — CHANGES_REQUESTED** [review]
In addition to what Ron pointed out, I found a few minor fixes if you want to accept them with the button below each one. Thanks!

**Comment by Thraka** [comment]
@paul1956 Sorry, I was on vacation. I'm back now. Please fix the merge conflicts and then I'll approve this PR. Thanks a bunch!

**Comment by paul1956** [comment]
@Thraka The resolve conflict button is disabled. I though I worked with @rpetrusha to resolve the .gitignore Issue and put back the original.

**Comment by Thraka** [comment]
@paul1956 Sorry for not noticing this earlier, but your copy of the repo you were working out of was a year out of date, which is causing all sorts of conflicts. For example, the current article has the latest version as **Visual Basic 15.8** yet your version lists **Visual Basic 15.5** as the latest (before you added 16). Additionally, the `.vb` file you edited was actually deleted from the repo about 5 months ago.

Again, I do apologize for not spotting this earlier, but it would be easier for you to start over with a fresh copy of the repo. You can do that by deleting your fork on github and recreating it with the fork button at the top of this repo. OR you can open a pull request from this repo into your own updating master. ([here is a link to do that](https://github.com/paul1956/docs/compare/master...dotnet:master?expand=0)) Then, create a **new** working branch (like you did with LineContinuationWithComments) and start fresh.

**Comment by paul1956** [comment]
@ThrakaThere is no VB 15.8 that I know of,  VB went from 15.5 to 16.

**Comment by paul1956** [comment]
From Roslyn Sosurce, notice no VisualBasic15_8, I think the change you are refering to as 15_8 is a performance improvement but does not change the language in any way.
```VB
    Public Enum LanguageVersion
        [Default] = 0
        VisualBasic9 = 9
        VisualBasic10 = 10
        VisualBasic11 = 11
        VisualBasic12 = 12
        VisualBasic14 = 14
        VisualBasic15 = 15
        VisualBasic15_3 = 1503
        VisualBasic15_5 = 1505
        VisualBasic16 = 1600

        Latest = Integer.MaxValue
    End Enum
        Friend Function GetLanguageVersion(feature As Feature) As LanguageVersion
            Select Case feature
                 ' lines deleted for simplicity
                Case Feature.LeadingDigitSeparator,
                    Feature.NonTrailingNamedArguments,
                    Feature.PrivateProtected
                    Return LanguageVersion.VisualBasic15_5

                Case Feature.UnconstrainedTypeParameterInConditional,
                    Feature.CommentsAfterLineContinuation
                    Return LanguageVersion.VisualBasic16
```

**Pull request closed without merging**